core/test/unit/models/tag_spec.js

```diff
@@ -16,7 +16,8 @@ describe('Unit: models/tag', function () {
         sandbox.restore();
     });
 
-    describe('SQL', function () {
+    // @NOTE: need to be regenerated once functional tests are passing
+    describe.skip('SQL', function () {
```

Comment:
`filter=count.posts:>=X` was never supported.

It was resulting in

> message: "select count(distinct tags.id) as aggregate from `tags` where `count`.`posts` >= 2 - ER_BAD_FIELD_ERROR: Unknown column 'count.posts' in 'where clause'",

Because this is a unit test and it only captures the generated SQL statement, it does succeed. Otherwise, the db would throw an error.

Only `include=count.posts` is supported.

Why is now throwing an error?
Because mongo-knex currently throws an error and has a logic bug [here](https://github.com/NexesJS/mongo-knex/blob/master/lib/convertor.js#L75). If we can't detect a relation, we need to assume that it's a column on the destination table. 

I will update mongo-knex and then update or remove this test. Need to see if this test has any value.

Reply:
Sounds perfect. We've put that error handling there as an experiment anyway, this case clearly shows why we should not have it there :)